ansible/playbooks/kolide-fleet.yml

```diff
@@ -67,3 +67,5 @@
   - name: Start fleet
     tags: fleet
     command: nohup /usr/local/bin/fleet serve --config /root/.kolide.yml > /var/log/fleet.log 2> /var/log/fleet.err < /dev/null &
+    async: 45
+    poll: 0
```

Comment:
We don't care about waiting for the command to finish?

Reply:
No, we don't, because it starts fleet and fleet just keeps going. If you wait for it to finish, ansible will never complete.